packages/rum-core/src/domain/contexts/accountContext.ts

```diff
@@ -12,6 +15,19 @@ export function startAccountContext(
     storeContextManager(configuration, accountContextManager, 'rum', CustomerDataType.Account)
   }
 
+  hooks.register(HookNames.Assemble, ({ eventType }): PartialRumEvent | undefined => {
+    const account = accountContextManager.getContext()
+
+    if (isEmptyObject(account) || !account.id) {
+      return
+    }
+
+    return {
+      type: eventType,
+      account: accountContextManager.getContext() as Account,
```

Comment:
Seems like you could reuse `account` here and avoid invoking `getContext()` again:
```suggestion
      account: account as Account,
```